homeassistant/components/tuya/light.py

```diff
@@ -361,6 +362,13 @@ def __init__(
                 None,
             )
 
+        # Determine color mode DPCode
+        if (
+            isinstance(description.color_mode, DPCode)
```

Comment:
As the typing suggests, this can only be DPCode or None, so we can do this without instance checking (instance is more expensive).

```suggestion
            description.color_mode is not None
```